src/domain/common/visual/visual.entity.ts

```diff
@@ -9,7 +9,7 @@ export class Visual extends AuthorizableEntity implements IVisual {
   @Column()
   name!: string;
 
-  @Column('text', { default: '' })
+  @Column('text', { nullable: false })
```

Comment:
Is text the right type for this? maybe varchar with long text length? But am not sure of the trade offs text vs varchar.

Reply:
https://stackoverflow.com/a/417184/11207901

> ...and the reality
That's what the standards say. For the reality, there was an article on [boutell.com](https://web.archive.org/web/20190902193246/https://boutell.com/newfaq/misc/urllength.html) (link goes to Internet Archive backup) that discussed what individual browser and server implementations will support. The executive summary is:
>> Extremely long URLs are usually a mistake. URLs over 2,000 characters will not work in the most popular web browsers. Don't use them if you intend your site to work for the majority of Internet users.

Reply:
Ok, then varchar with LONG_TEXT_LENGTH?